GLAA.Domain/Models/GLAAUser.cs

```diff
@@ -1,12 +1,27 @@
-﻿using Microsoft.AspNetCore.Identity;
+﻿using System;
+using Microsoft.AspNetCore.Identity;
 using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace GLAA.Domain.Models
 {
-    public class GLAAUser : IdentityUser
+    public class GLAAUser : IdentityUser, IAddressable
     {
-        public string FullName { get; set; }
+        public string Title { get; set; }
+        public string FirstName { get; set; }
+        public string MiddleName { get; set; }
+        public string LastName { get; set; }
+
+        public string FullName => (string.IsNullOrEmpty(Title) ? string.Empty : $"{Title} ") +
```

Comment:
Are the null or empty checks here to cater for instantiation rather than none presence of those fields? (cos they're mandatory aren't they?) in which case +1!

Reply:
Mostly for instantiation, although `Title` and `MiddleName` are optional so need to be catered for anyway.